internal/objects/daemonset/daemonset_test.go

```diff
@@ -163,8 +162,8 @@ func TestDesiredDaemonSet(t *testing.T) {
 		NetworkType: operatorv1alpha1.LoadBalancerServicePublishingType,
 	}
 	cntr := objcontour.New(cfg)
-	testContourImage := config.DefaultContourImage
-	testEnvoyImage := config.DefaultEnvoyImage
+	testContourImage := "ghcr.io/projectcontour/contour:test"
```

Comment:
+1 for this kind of change, using some test data for these rather than constants etc. that may be "real"

Reply:
Cool, yeah I had to do this to break the import cycle but there was no good reason to use the actual constants here.